src/ruby/README.md

```diff
@@ -4,31 +4,28 @@ gRPC Ruby
 A Ruby implementation of gRPC.
 
 Status
--------
+------
 
 Alpha : Ready for early adopters
 
-INSTALLATION PREREQUISITES
---------------------------
+PREREQUISITES
+-------------
 
-This requires Ruby 2.1, as the RPC API surface uses keyword args.
+- Ruby 2.x. The gRPC API uses keyword args.
+- [homebrew][] on Mac OS X, [linuxbrew][] on Linux.  These simplify the installation of the gRPC C core.
```

Comment:
Can you modify link for linuxbrew to point to https://github.com/Homebrew/linuxbrew#installation